frontend/src/model/beta/Releases.tsx

```diff
@@ -1,18 +1,32 @@
 import { Box, Button, Stack } from '@mui/material'
 import { useEffect, useState } from 'react'
 
+import { useGetReleasesForModelId } from '../../../actions/release'
+import { ModelInterface, ReleaseInterface } from '../../../types/types'
+import EmptyBlob from '../../common/EmptyBlob'
+import Loading from '../../common/Loading'
 import DraftNewReleaseDialog from '../DraftNewReleaseDialog'
 import ModelReleaseDisplay from '../ModelReleaseDisplay'
 
-export default function Releases({ model }: { model: any }) {
+export default function Releases({ model }: { model: ModelInterface }) {
   const [latestRelease, setLatestRelease] = useState<string>('')
   const [openDraftNewRelease, setOpenDraftNewRelease] = useState(false)
 
+  const { releases, isReleasesLoading } = useGetReleasesForModelId(model.id)
+
+  const sortByReleaseVersionDescending = (a: ReleaseInterface, b: ReleaseInterface) => {
+    if (a.semver < b.semver) return 1
+    if (b.semver > a.semver) return -1
+    return 0
+  }
+
+  const sortedReleases = releases.sort(sortByReleaseVersionDescending)
```

Comment:
Sorry another thing I should've spotted the first time around. Can we wrap this with a `useMemo` so we're not sorting on every render?
```
const sortedReleases = useMemo(() => releases.sort(sortByReleaseVersionDescending), [releases])
```